Topology/README.md

```diff
@@ -28,10 +29,59 @@ The following are optional fields and default values (option, more a list of pos
     - `doi` - A doi reference for the molecule.
 
 Other possible quantities:
-  - Bonds - Holding data for MM computations
   - Basis Sets per atom 
   - `fix_com` (bool) - whether to adjust to the molecule to the COM or not
   - `fix_orientation` (bool) - whether to rotate the molecule to a standard orientation or not
   - label (list of str) - Per-atom labels which may be seperate from fragments
   - Extend the `real` quantitity to cover real, ghost, absent, qm/mm region, etc.
   - EFP quantities `fragment_types`, `coordinate_hints`. This is an example and likely not part of the spec. How would we handle this? 
+
+# Connectivity
+
+Connectivity describes relationships between lists of atoms and associated metadata: typically, this is between a pair of atoms (a bond). Connectivity is supported across periodic boundaries.
+
+Each connection between `M` atoms is defined as:
+
+* `M`, number of atomic sites involved, `M` is 2 for connections between pairs of atoms (e.g. bonds)
+* `indices`, an ordered list of length `N` of atomic indexes defining the connection, length is multiple of `M`
+* `images`, (optional, only relevant if a unit cell lattice is defined) an ordered list of length `3N` describing the periodic image of the connected atom -- by default, this is assumed to be (0,0,0), meaning that the connected atom is in the origin image, and first index is set to (0,0,0) by convention
+* a dictionary of metadata, where each value is a list of length `N` (for example, a key could be "bond_type" and values could be "single", "double", etc.)
+
+As an example:
+
+`indices`: [0, 1, 5, 6]
+`images`: [0,0,0, 0,0,0, 0,0,0, 1,0,0]
+
+defines two bonds, one between atoms 0 and 1 and one between atoms 5 and 6. In this case, the atom 6 is in periodic image (1,0,0).
+
+For metadata, there are a few pre-defined keys with strict meanings:
+
+* `bond_type` can take values `single`, `double`, `triple` [to replace -- this is just an example], only suitable for connections involving pairs of atoms, this is useful for GUI/visualization tools
```

Comment:
What about order, and use an integer rather than a string, or are there other types being envisaged?

Reply:
Order sounds sensible! `bond_type` was just an example to get the conversation started: some pre-defined keys would be good, I'm (personally) agnostic as to what these keys should be.

Reply:
But order and an integer value sounds good.

Reply:
Sounds like geometry should be changed following discussions from yesterday. I can make a pull request.